cgp/cartesian_graph.py

```diff
@@ -397,11 +433,11 @@ def to_sympy(self, simplify: Optional[bool] = True) -> List["sympy_expr.Expr"]:
             sympy_exprs.append(sympy.sympify(s, evaluate=False))
 
         if not simplify:
-            return sympy_exprs
+            return possibly_unpack(sympy_exprs)
         else:  # simplify expression if desired and possible
             for i, expr in enumerate(sympy_exprs):
                 try:
                     sympy_exprs[i] = expr.simplify()
                 except TypeError:
                     RuntimeWarning(f"SymPy could not simplify expression: {expr}")
-            return sympy_exprs
+            return possibly_unpack(sympy_exprs)
```

Comment:
The return statements can be outside of the if/else-branch and then you only need what is currently the else branch,

Reply:
like so:
```python
if simplify:
    for i, expr in enumerate(sympy_exprs):
                try:
                    sympy_exprs[i] = expr.simplify()
                except TypeError:
                    RuntimeWarning(f"SymPy could not simplify expression: {expr}")
return possibly_unpack(sympy_exprs)
```